entrypoint.sh

```diff
@@ -246,6 +247,14 @@ then
    fruit:wipe_intentionally_left_blank_rfork = yes
    fruit:delete_empty_adfiles = yes" > /etc/samba/smb.conf
   fi
+  if [ "${IGNORE_DOS_ATTRIBUTES}" == "true" ]
+  then
+    echo "   store dos attributes = no
+   map hidden = no
```

Comment:
What are the `map *` settings needed for?  Curious if you have a reference I could check.

Reply:
If the DOS attributes are not stored in an extended attribute (i.e. `store dos attributes = no`) Samba might try to map DOS attributes to linux permissions bits. Per default actually only the archive bit for whatever reason ...
By setting the `map*` setting to `no` Samba will not try to do so.
See updated `README.md` for more details.